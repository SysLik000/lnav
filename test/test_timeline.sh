#! /bin/bash

export YES_COLOR=1

export TZ=UTC

touch -t 200711030923 ${srcdir}/logfile_glog.0
run_cap_test ${lnav_test} -n \
    -c ";UPDATE all_logs set log_opid = 'test1' where log_line in (1, 3, 6)" \
    ${test_dir}/logfile_glog.0

run_cap_test ${lnav_test} -n \
    -c ";UPDATE all_logs set log_opid = 'test1' where log_line in (1, 3, 6)" \
    -c ":hide-fields log_opid" \
    ${test_dir}/logfile_glog.0

run_cap_test ${lnav_test} -n \
    -c ";UPDATE all_logs set log_opid = 'test1' where log_line in (1, 3, 6)" \
    -c ':switch-to-view timeline' \
    ${test_dir}/logfile_glog.0

run_cap_test ${lnav_test} -n \
    -c ";UPDATE all_logs set log_opid = '1234' where log_line in (1, 3, 6)" \
    -c ";UPDATE all_opids SET description = 'test-1' WHERE opid = '1234'" \
    -c ";UPDATE all_logs set log_opid = '5678' where log_line = 2" \
    -c ";UPDATE all_opids SET description = 'test-2' WHERE opid = '5678'" \
    -c ':switch-to-view timeline' \
    ${test_dir}/logfile_glog.0

run_cap_test ${lnav_test} -n \
    -c ";UPDATE all_logs set log_opid = '1234' where log_line in (1, 3, 6)" \
    -c ";UPDATE all_opids SET description = 'test-1' WHERE opid = '1234'" \
    -c ";UPDATE all_logs set log_opid = '5678' where log_line = 2" \
    -c ";UPDATE all_opids SET description = 'test-2' WHERE opid = '5678'" \
    -c ";SELECT * FROM all_opids" \
    ${test_dir}/logfile_glog.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    ${test_dir}/logfile_generic.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':filter-in CdysLK1XpcrXOpVDuh' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':filter-out CdysLK1XpcrXOpVDuh' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':hide-file *' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':close *' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':hide-lines-before 2011-11-03 00:19:37' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    -c ':hide-lines-after 2011-11-03 00:20:30' \
    ${test_dir}/logfile_bro_http.log.0

run_cap_test ${lnav_test} -n \
    -c ':switch-to-view timeline' \
    ${test_dir}/logfile_strace_log.1
