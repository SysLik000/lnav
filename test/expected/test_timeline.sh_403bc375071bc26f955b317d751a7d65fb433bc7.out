[1m[4m[35m   Duration   [0m[4m|[0m[4m [0m[1m[4m[31mâœ˜[0m[4m[33mâ–²[0m[4m [0m[4m|[0m[1m[4m[35m Operation[0m[4m  [0m[4m|[0m[4m  37s    [0m[4m|[0m[4m1m13s    [0m[4m|[0m[4m1m49s[0m
[32m           844[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCwF[0m[7m[32ms[0m[32m1P2UcUdlSxD2La[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCoX[0m[7m[32m7[0m[32mzA3OJKGUOSCBY2[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCdr[0m[1m[7m[32mf[0m[1m[32mXZ1NOFPEawF218[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCJw[0m[1m[7m[32mU[0m[1m[32mi9bdB9c1lLW44[0m[1m[32m  [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCJx[0m[7m[32mS[0m[32mUgkInyKSHiju1[0m[32m  [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCT0[0m[7m[32mJ[0m[32mIh479jXIGt0Po1[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC6Q[0m[1m[7m[32m4[0m[1m[32mVm14ZJIlZhsXqk[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m         1s070[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCtg[0m[1m[7m[32mx[0m[1m[32mRAqDLvrRUQdqe[0m[1m[32m  [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCdy[0m[7m[32ms[0m[32mLK1XpcrXOpVDuh[0m[32m [0m[32m192.168.2.76[0m
[32m           200[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC6n[0m[7m[32mS[0m[32moj1Qco9PGyslz6[0m[32m [0m[32m192.168.2.76[0m
[1m[32m        23s044[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCN5[0m[1m[7m[32mhnY3x5[0m[1m[32m1j6Hr1v4[0m[1m[32m  [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCdZ[0m[1m[7m[32mU[0m[1m[32mPH2DKOE7zzCLE3[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m        32s388[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCmW[0m[7m[32mpC33jXuKp[0m[32mXNLcie[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCsBg[0m[7m[32mi[0m[32mE1WmGP4Yo749h[0m[32m [0m[32m192.168.2.76[0m
[1m[32m           657[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCYfH[0m[1m[7m[32my[0m[1m[32mC28tAhkLYkXB7[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCtAN[0m[1m[7m[32mm[0m[1m[32mVrHYMtkWqPE5[0m[1m[32m  [0m[1m[32m192.168.2.76[0m
[32m           647[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCSTH[0m[7m[32m8[0m[32mn1O1nv0ztxNQd[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC4uD[0m[7m[32mK[0m[32mU5tpeRU9Su19[0m[32m  [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCEh6Ka[0m[1m[7m[32m2[0m[1m[32mHInkNSH01L2[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m         4s840[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCjPGiy[0m[1m[7m[32m1[0m[1m[32m3ncXKxU765j[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m         6s895[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCPoz7N[0m[7m[32mUp[0m[32mXISemlNSd[0m[32m  [0m[32m192.168.2.76[0m
[32m         6s837[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCt6ixh[0m[7m[32m35[0m[32my9AEr7J7o9[0m[32m [0m[32m192.168.2.76[0m
[1m[32m         6s889[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCaEFHq[0m[1m[7m[32m2H[0m[1m[32mVQ5iGJQiD9[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m         6s875[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCjinlH[0m[1m[7m[32m2f[0m[1m[32mzDtvzI9637[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m         6s884[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCedw7H[0m[7m[32m3d[0m[32mdE2yLiLoXc[0m[32m [0m[32m192.168.2.76[0m
[32m         6s917[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCAUlC2[0m[7m[32m49[0m[32msvUfE6q0g3[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCIJIDL[0m[1m[7m[32m1[0m[1m[32mULo4HpT24Gl[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m           016[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCLsqp4[0m[1m[7m[32m1[0m[1m[32mRLUd83arUQb[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m           166[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCbCciH[0m[7m[32m1[0m[32m1995WKkobR1[0m[32m [0m[32m192.168.2.76[0m
[32m           457[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mClcvKE[0m[7m[32m1[0m[32mdqsEFQu46m9[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCaP2Lp[0m[1m[7m[32mL[0m[1m[32mGvsmX7yJO[0m[1m[32m   [0m[1m[32m192.168.2.76[0m
[1m[32m           043[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCNbPns[0m[1m[7m[32m4[0m[1m[32mmOMGgjI8Ele[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m           557[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC185u7u[0m[7m[32m9[0m[32mQ4qhJPhzl[0m[32m  [0m[32m192.168.2.76[0m
[32m           129[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCKzjfhs[0m[7m[32mJ[0m[32m8vrn2rrfg[0m[32m  [0m[32m192.168.2.76[0m
[1m[32m           562[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCiIjAe1[0m[1m[7m[32mn[0m[1m[32m5MnPOVpQ9f[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCGv2Tp4[0m[1m[7m[32mN[0m[1m[32mgt8MmKmVRd[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m         7s071[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC5DisEM[0m[7m[32mFU[0m[32m77Wk9Kae[0m[32m  [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCs5yEZ3E[0m[7m[32mL[0m[32mZTeuTOsP4[0m[32m [0m[32m192.168.2.76[0m
[1m[32m         4s667[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCu4gIx1B[0m[1m[7m[32mD[0m[1m[32mNtGOl7Ht2[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m         1s288[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCRgW2I2z[0m[1m[7m[32mo[0m[1m[32m3SInm6iT8[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCWJhMU2c[0m[7m[32mT[0m[32mLEnseTmCb[0m[32m [0m[32m192.168.2.76[0m
[32m           042[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCejI402r[0m[7m[32mK[0m[32mGtdBXij4f[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC2KnU34G[0m[1m[7m[32mc[0m[1m[32mVV6amo8va[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC5vx4911[0m[1m[7m[32mi[0m[1m[32mSMAJuShFd[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCbUCgw1D[0m[7m[32mr[0m[32mIGcXzONB7[0m[32m [0m[32m192.168.2.76[0m
[32m           502[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC96j2X1D[0m[7m[32mi[0m[32mxgLTj2Oi8[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCYYyja3FF[0m[1m[7m[32mN[0m[1m[32mEnftw3K6[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31mâ–ˆ[0m[1m[33m [0m[1m[32m  [0m[1m[32mCBHHuR1[0m[1m[7m[32mx[0m[1m[32mFnm5C5CQBc[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCD1jfU3p9[0m[7m[32ma[0m[32mbEm77mzf[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC0K9DaoPF[0m[7m[32mk[0m[32mfnzwlZa[0m[32m  [0m[32m192.168.2.76[0m
[1m[32m           616[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCbQAWi3GX2bC[0m[1m[7m[32mm[0m[1m[32mX5L56[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m           298[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCd8s2R3OGDgk[0m[1m[7m[32mh[0m[1m[32mnvSu9[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m           205[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCBeaXe4Iyj1g[0m[7m[32mX[0m[32md2Iq[0m[32m  [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCmWpSw3VtjiA[0m[7m[32mc[0m[32meBCwf[0m[32m [0m[32m192.168.2.76[0m
[1m[32m      1m12s201[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCbNCgO1MzloH[0m[1m[7m[32mRNeY4f[0m[1m[7m[32m 192.168.2.76[0m
[1m[32m           315[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCX1GjC4vn52U[0m[1m[7m[32mY[0m[1m[32m1uDv6[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCaPClb1Bf0Rr[0m[7m[32mR[0m[32mGtyWi[0m[32m [0m[32m192.168.2.76[0m
[32m        35s642[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCibfNy1QQW4I[0m[7m[32mmDWRq5[0m[7m[32m 192[0m[32m.168.2.76[0m
[1m[32m      1m11s547[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCTRXSR3blXJE[0m[1m[7m[32m5ZE7Ij[0m[1m[7m[32m 192.168.2.76[0m
[1m[32m      1m11s536[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCnGze54kQWWp[0m[1m[7m[32mKqrrZ4[0m[1m[7m[32m 192.168.2.76[0m
[32m        12s337[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC3TZMB4CrUwY[0m[7m[32mfkGJ[0m[32my1[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCK957ERTz8lB[0m[7m[32my[0m[32mcly4[0m[32m  [0m[32m192.168.2.76[0m
[1m[32m         1s309[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCO5QKYQkcSdxQFA[0m[1m[7m[32m3[0m[1m[32m5[0m[1m[32m  [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCurHpb1TGZOktTR[0m[1m[7m[32mN[0m[1m[32mP1[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCuUKOQ1R3CqKBge[0m[7m[32mT[0m[32mdf[0m[32m [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC3xkHgJnzZszVSTp[0m[7m[32mi[0m[32m  [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCMrjgF2XLmRh9C9T[0m[1m[7m[32mR[0m[1m[32m4[0m[1m[32m [0m[1m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC2vQ8sVgyADHjtEda[0m[1m[32m  1[0m[1m[7m[32m9[0m[1m[32m2.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCD69521bDXIAb4IkW[0m[32m  1[0m[7m[32m9[0m[32m2.168.2.76[0m
[32m         6s648[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCC3vUI3gFB04zLvWRa[0m[32m 1[0m[7m[32m92[0m[32m.168.2.76[0m
[1m[32m         2s666[0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC7Krri4g9tZfHniGXh[0m[1m[32m 192[0m[1m[7m[32m.[0m[1m[32m168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCmxyBl2c8XAMTuHEk4[0m[1m[32m 192[0m[1m[7m[32m.[0m[1m[32m168.2.76[0m
[32m           499[0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCSvs6v26bQqFylkk6l[0m[32m 192[0m[7m[32m.[0m[32m168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC4pHul1H3OeWYz7o7i[0m[32m 192.[0m[7m[32m1[0m[32m68.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mC7Lcvr4vsTf6eYpBva[0m[1m[32m 192.[0m[1m[7m[32m1[0m[1m[32m68.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCV8faD4L1sLL5kDwN9[0m[1m[32m 192.[0m[1m[7m[32m1[0m[1m[32m68.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCxyAKs10ppnHFP6O8i[0m[32m 192.[0m[7m[32m1[0m[32m68.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC6MrHk2C7rLuJqhjsg[0m[32m 192.[0m[7m[32m1[0m[32m68.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCvfUrT2DgYXXoZw9Ah[0m[1m[32m 192.[0m[1m[7m[32m1[0m[1m[32m68.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCBX0254QJoklXNbvv2[0m[1m[32m 192.[0m[1m[7m[32m1[0m[1m[32m68.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mC6Ym6jvMgikT0xTTc[0m[32m  [0m[32m192.168.2.76[0m
[32m              [0m[32m  [0m[1m[31m [0m[33m [0m[32m  [0m[32mCRNn9f1zKNlzHSM5pa[0m[32m [0m[32m192.168.2.76[0m
[1m[32m              [0m[1m[32m  [0m[1m[31m [0m[1m[33m [0m[1m[32m  [0m[1m[32mCJLgi92kpp2gLgGTE5[0m[1m[32m [0m[1m[32m192.168.2.76[0m
